src/main/java/duke/tasks/Deadline.java

```diff
@@ -0,0 +1,31 @@
+package duke.tasks;
+
+public class Deadline extends Task {
+
+    protected String by;
+
+    public Deadline(String description, Boolean isDone, String by) {
+        super(description, isDone);
+        this.by = by;
+    }
+
+    @Override
+    public String getTaskType() {
+        return "D";
```

Comment:
magic string? please refer to https://nus-cs2113-ay2021s1.github.io/website/se-book-adapted/chapters/codeQuality.html#guideline-name-well